integrationTests/chainSimulator/relayedTx/relayedTx_test.go

```diff
@@ -255,7 +263,199 @@ func TestRelayedTransactionInMultiShardEnvironmentWithChainSimulatorScCalls(t *t
 	}
 }
 
-func startChainSimulator(t *testing.T) testsChainSimulator.ChainSimulator {
+func TestFixRelayedMoveBalanceWithChainSimulator(t *testing.T) {
+	if testing.Short() {
+		t.Skip("this is not a short test")
+	}
+
+	expectedFeeScCall := "815285920000000"
+	t.Run("sc call", testFixRelayedMoveBalanceWithChainSimulatorScCall(expectedFeeScCall, expectedFeeScCall))
```

Comment:
so for this test the expected fee, before and after fix, is the same?

Reply:
yes, the fix should not influence sc calls, but only move balance